Formula/libideviceactivation.rb

```diff
@@ -0,0 +1,26 @@
+class Libideviceactivation < Formula
+  desc "Library to handle the activation process of iOS devices"
+  homepage "https://www.libimobiledevice.org/"
+  head "https://github.com/libimobiledevice/libideviceactivation.git"
```

Comment:
Homebrew ships software that has working, stable releases. Head is an option for some users, but should not be the main installation mechanism.